app/admin/topics.rb

```diff
@@ -1,3 +1,23 @@
 ActiveAdmin.register Topic do
-  permit_params :name
+  form do |f|
+    f.inputs 'Topic' do
+      f.input :name
+      f.input :image, as: :file
+    end
+    f.actions
+  end
+  permit_params :name, :image
```

Comment:
WDTY about moving `permit_params :name, :image` to the top to find it more easily?

Reply:
Yes, It's fine for me, I will change it